tests/test_helpers.py

```diff
@@ -32,6 +32,35 @@
       "--authors", "sbillinge", "ashaaban", "--abstract", "the earth is round as seen from mars",
       "--title", "On the roundness of the Earth", "--status", "in-prep"],
      "2006as_mars has been added in presentations\n"),
+    (["helper", "l_progress", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+      "    description: My first projectum\n"
+     ),
+    (["helper", "l_progress", "-v", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+      "    description: My first projectum\n"
+      "    log_url: https://docs.google.com/document/d/1YC_wtW5Q\n"
+      "    team:\n"
+      "        group_members: ascopatz\n"
+      "        collaborators: aeinstein, pdirac\n"
+      "    deliverable:\n"
+      "        due_date: 2021-05-05\n"
```

Comment:
this is not needed as the deliverable due date is also the project due date which is above

Reply:
Done.